src/components/helpPopup/helpPopup.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
```

Comment:
let's move it to components/chart and call it chartHelp

Reply:
done